.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,10 @@
+<!--
+Note that this is the issue tracker for OpenShift Origin.
+
+OpenShift Installer issues should be reported at https://github.com/openshift/openshift-ansible/issues
+Documentation issues is better to report at https://github.com/openshift/openshift-docs/issues
```

Comment:
What is this page for? I wonder if there is a better mechanism for users to report documentation issues, if this page is for end users. Not sure how closely the openshift-docs/issues is monitored.

Reply:
@mburke5678 This is a template for a new issues. See the following link for details:
- https://help.github.com/articles/creating-an-issue-template-for-your-repository/
- https://github.com/blog/2111-issue-and-pull-request-templates

> I wonder if there is a better mechanism for users to report documentation issues, if this page is for end users.

I don't know about such mechanism. Time to time, users reporting documentation issues here, but it happens not every day, so if you object against this paragraph, I will remove it.

Reply:
@php-coder Please leave the paragraph in. That is great!